### PR TITLE
add postgresql to database module

### PR DESCRIPTION
**Pull request by hektor1966**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -111,4 +177,45 @@ locals {
     for mysql_flexible_server_firewall_rule in keys(var.mysql_flexible_server_firewall_rule) :
     mysql_flexible_server_firewall_rule => merge(local.default.mysql_flexible_server_firewall_rule, var.mysql_flexible_server_firewall_rule[mysql_flexible_server_firewall_rule])
   }
+
+  /**
+    compare and merge custom and default values
+  */
+  postgresql_flexible_server_values = {
```
Comment (michaelamattes) [inline variables.tf:184]: don't duplicate the compare block again so please move this to line 147.

File: variables.tf [hunk 2]
```diff
@@ -111,4 +177,45 @@ locals {
...
+    for postgresql_flexible_server in keys(var.postgresql_flexible_server) :
+    postgresql_flexible_server => merge(local.default.postgresql_flexible_server, var.postgresql_flexible_server[postgresql_flexible_server])
+  }
+
+  /**
+    deep merge of all custom and default values
+  */
+  postgresql_flexible_server = {
```
Comment (michaelamattes) [inline variables.tf:192]: don't duplicate the compare block again so please move this to line 180

**Comment by michaelamattes** [comment]
@hektor1966  Please change the MR topic and remove your internal ticket ID as this is only your internal tracking and not a ticket accessible from anywhere in Github

File: variables.tf (hunk 1 above)
Comment (hektor1966) [reply variables.tf]: ok, done

File: variables.tf (hunk 2 above)
Comment (hektor1966) [reply variables.tf:192]: as discussed, is obsolete

**Review by eckes-granini-support — APPROVED** [review]

**Review by michaelamattes — APPROVED** [review]

**Pull request merged**